### PR TITLE
+87

### PR DESCRIPTION
**Pull request by shkwok**
on9 take mine

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Hello World!
 HIHIHI
+87
```
Comment (dicksonhk) [inline README.md]: why + 87??
I think +99 is better :o) :new_moon_with_face:

Comment (shkwok) [inline README.md]: because u are 87

Comment (dicksonhk) [inline README.md]: :joy:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 Hello World!
```
Comment (dicksonhk) [inline README.md]: Hello world is the best :sunglasses:

**Pull request merged**